pkg/sql/schema_changer_test.go

```diff
@@ -1737,6 +1737,64 @@ CREATE TABLE t.test (
 	}
 }
 
+// This test check whether we could add a same column name when this column is in DELETE_ONLY state.
```

Comment:
// This test checks whether a column can be added using the name of a column that has just been dropped.